bin/tronctl

```diff
@@ -251,6 +252,46 @@ def request(url: str, data: Dict[str, Any], headers=None, method=None) -> bool:
 
 def event_publish(args):
     for event in args.id:
+        # trying to publish a job run/action run id will likely print multiple warnings
+        # since the conditions are somewhat overlapping - only print the first one
+        warning_printed = False
+        split_event = event.split(".")
+        # first, let's try to catch folks trying to publish a job run or action run id as a trigger
+        # these will look something like NAMESPACE.JOB_NAME.RUN_NUMBER or NAMESPACE.JOB_NAME.RUN_NUMBER.ACTION_NAME
+        # i.e., if the 3rd element is an integer, it's one of these
+        if len(split_event) >= 3:
```

Comment:
i don't really love these len() shenanigans

but i'm not sure if doing something like
```Python
try:
    # try to validate that we can split into the expected number of chunks
    maybe_namespace, maybe_job, maybe_action, maybe_trigger_name, maybe_trigger_value = event.split(".")
    try:    
        # catch folks trying to publish a job/action id
        # since we'd have a number where we'd expect the 
        # action name
        int(maybe_action)
        ... # warn about likely invalid trigger id
    except ValueError:
        pass  # we should have a valid trigger here
except ValueError:
    ... # warn about likely invalid trigger id (too short or too long)
```
would be preferable to folks?